src/exceptions.jl

```diff
@@ -0,0 +1,7 @@
+struct LayerNotFoundException <: Exception
+	var::String
+end
+
+function Base.showerror(io::IO, e::LayerNotFoundException)
+	println(io, typeof(e), ": ", e.var)
+end
```

Comment:
Should use spaces instead of tabs. Seems to be just with new files

Reply:
Arg, I forgot to check that.
Good catch